.circleci/config.yml

```diff
@@ -376,9 +376,9 @@ jobs:
                   key: v0.4-code_quality-{{ checksum "setup.py" }}
                   paths:
                       - '~/.cache/pip'
-            - run: black --check examples tests src utils
-            - run: isort --check-only examples tests src utils
-            - run: flake8 examples tests src utils
+            - run: black --check examples research_projects tests src utils
```

Comment:
would `research` suffice? `research_projects` is somewhat long IMHO

Reply:
in other files you also refer to `research examples` and `research-examples` - this is confusing.